src/utils/tokens.ts

```diff
@@ -239,10 +239,10 @@ const tokens = {
   },
 
   /**
-   * Get the HTR value of the fee to create a NFT
-   * @returns {number} The fee to create a NFT
+   * Get the HTR value of the fee to add a data script output
+   * @returns {number} The fee to have a data script output
    */
-  getNFTCreationFee(): number {
+  getDataScriptOutputFee(): number {
```

Comment:
issue(if-minor): This may be used by the other wallets.

If it is being used we should remember to bump the lib and make this change on the wallets after this is released

Reply:
I thought we would be using it in the wallets but we actually don't. We have method in the wallet utils but we should change to use the lib.

https://github.com/HathorNetwork/hathor-wallet/blob/master/src/utils/tokens.js#L111